### PR TITLE
[PUBDEV-3847] added new R test, it requires the custom CSV (smalldata…

### PR DESCRIPTION
**Pull request by michal-raska**
Added new R test, which should fail (although not always, it is a bit indeterministic) It is aimed to resolve the `java.lang.AssertionError: Coldata 653.3299798300001 out of range` like errors

### CODE REVIEW

**Comment by mmalohlava** [comment]
@michalkurka do you want to mark it as NOPASS for now?

**Comment by navdeep-G** [comment]
Looks good, I would make it NOPASS for now.

**Comment by michalkurka** [comment]
We can make it NOPASS for now.

**Comment by michal-raska** [comment]
@h2o-ops please test!

File: h2o-r/tests/testdir_algos/gbm/runit_GBM_coldata_assertion_error.R [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    max_depth <- 6
+    min_rows <- 5
+    learn_rate <- 0.1
+    sample_rate <- 0.8
+    col_sample_rate_per_tree <- 0.6
+    nfolds <- 2
+    min_split_improvement <- 1e-04
+
+    model <- h2o.gbm(x = myX, y = myY, training_frame = data, model_id = "amodel", ntrees = ntrees,
+    max_depth <- max_depth , min_rows = min_rows, learn_rate = learn_rate,
```
Comment (mmalohlava) [inline h2o-r/tests/testdir_algos/gbm/runit_GBM_coldata_assertion_error.R]: `=` should be use everywhere, no?

Comment (navdeep-G) [inline h2o-r/tests/testdir_algos/gbm/runit_GBM_coldata_assertion_error.R]: `<-` is used for assignment, but function calls use `=`

Comment (navdeep-G) [inline h2o-r/tests/testdir_algos/gbm/runit_GBM_coldata_assertion_error.R]: Its just R practices

Comment (mmalohlava) [inline h2o-r/tests/testdir_algos/gbm/runit_GBM_coldata_assertion_error.R]: and also there is semantics difference how it deals with side-effects, `=` is much more safer in calls

Comment (navdeep-G) [inline h2o-r/tests/testdir_algos/gbm/runit_GBM_coldata_assertion_error.R]: To be honest, I am not a fan of `<-`. Just requested it for consistency sake. We can forgo it from now on.

Comment (michal-raska) [inline h2o-r/tests/testdir_algos/gbm/runit_GBM_coldata_assertion_error.R]: thank you very much guys, will change it in the morning

**Comment by michal-raska** [comment]
@h2o-ops please test!

**Comment by mmalohlava** [comment]
@michal-raska it should fail, no? I can see only green status (which is good, but the test is not marked by NOPASS)

**Comment by michal-raska** [comment]
@mmalohlava it should, but that is the problem with the test, it fails only sometimes (as stated also in the discussion below the JIRA). I have run it several times, but it didn't fail once. Maybe it is fixed?

**Comment by michal-raska** [comment]
@mmalohlava I've run it 100 times if for loop on localhost, still no FAIL, nor is it failing in Jenkins. I believe it depends strongly which machine runs the test as we have discussed it yesterday

**Comment by maurever** [comment]
After discussion with @michalkurka, I close this PR. I created the new [PR](https://github.com/h2oai/h2o-3/pull/3087), which adds the new test to rel-xia branch (but the test still doesn't fail).

**Pull request closed without merging**